base_vat_optional_vies/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================================
+Optional validation of VAT via VIES
+===================================
+
+This module extends base_vat module features allowing to know if VIES
+validation was passed or not.
+
+Then you can use "VIES validation passed" field in order to show VAT ID with
+or without country preffix in invoices, for instance.
+
+
+Installation
+============
+
+To install this module, you need vatnumber python library:
```

Comment:
This is already a requirement of Odoo itself, so no need to mention: https://github.com/odoo/odoo/blob/9.0/requirements.txt#L38